src/js/msp/MSPHelper.js

```diff
@@ -731,6 +731,10 @@ MspHelper.prototype.process_data = function(dataHandler) {
             case MSPCodes.MSP_API_VERSION:
                 FC.CONFIG.mspProtocolVersion = data.readU8();
                 FC.CONFIG.apiVersion = data.readU8() + '.' + data.readU8() + '.0';
+                if (!FC.CONFIG.apiVersion || FC.CONFIG.apiVersion === 'null.null.0') {
+                    FC.CONFIG.apiVersion = '0.0.0';
+                }
+                console.log(FC.CONFIG.apiVersion);
```

Comment:
This seems an ugly hack for the problem.
This only happens with this two parameters? Or the `mspProtocolVersion` is bad too when the problem happens?

If both are bad, maybe simply the MSP connection is bad at this moment for some reason, if only the api version values are bad then maybe the FC is sending bad the value for some reason.

Did you verify if the `mspProtocolVersion` was bad?

Reply:
I have seen the FC has the api version defined as constants (define) so it is difficult that it fails. Something must be wrong with the MSP connection on our part, but I've never seen a problem with the reading of this in other parts of the Configurator.

Reply:
Not sure about this either as former code relies on this value. But still the ugly hack is working. There seems to be something strange with the introduction of modules.

Reply:
The error also can be found at:
```
PortHandler - Removed: [{"path":"/dev/ttyACM0","displayName":"Betaflight STM32F411"}]
DevTools failed to load SourceMap: Could not load content for chrome-extension://ajdoecjhplhoahggfmgflfnlechcljoc/node_modules/jquery-textcomplete/dist/dist/jquery.textcomplete.min.map: System error: net::ERR_FILE_NOT_FOUND
port_handler.js:159 PortHandler - Found: [{"path":"/dev/ttyACM1","displayName":"Betaflight STM32F411"}]
serial_backend.js:69 serial: connecting to: /dev/ttyACM1
serial.js:110 serial: connection opened with ID: 4 , Baud: 115200
serial_backend.js:249 Requesting configuration data
main.html#:1 Error in event handler: TypeError: Invalid Version: null.null.0
    at new l (chrome-extension://ajdoecjhplhoahggfmgflfnlechcljoc/node_modules/semver-min/umd/index.js:1:4884)
    at I (chrome-extension://ajdoecjhplhoahggfmgflfnlechcljoc/node_modules/semver-min/umd/index.js:1:8498)
    at Object.L [as gte] (chrome-extension://ajdoecjhplhoahggfmgflfnlechcljoc/node_modules/semver-min/umd/index.js:1:8702)
    at MspHelper.process_data (chrome-extension://ajdoecjhplhoahggfmgflfnlechcljoc/js/msp/MSPHelper.js:772:28)
    at chrome-extension://ajdoecjhplhoahggfmgflfnlechcljoc/js/msp.js:229:13
    at Array.forEach (<anonymous>)
    at Object.notify (chrome-extension://ajdoecjhplhoahggfmgflfnlechcljoc/js/msp.js:228:24)
    at Object._dispatch_message (chrome-extension://ajdoecjhplhoahggfmgflfnlechcljoc/js/msp.js:219:14)
    at Object.read (chrome-extension://ajdoecjhplhoahggfmgflfnlechcljoc/js/msp.js:186:22)
    at read_serial (chrome-extension://ajdoecjhplhoahggfmgflfnlechcljoc/js/serial_backend.js:590:13)
msp.js:347 MSP data request timed-out: 4 direction: 1 tab: landing
```

Removed the hack for now:
```
    if (!FC.CONFIG.apiVersion || FC.CONFIG.apiVersion === 'null.null.0') {
        // something is wrong
        FC.CONFIG.apiVersion = '0.0.0';
    }
```
Without the hack the user has to restart configurator.

Reply:
Made an issue for this: https://github.com/betaflight/betaflight-configurator/issues/2498

Reply:
Created an issue for this https://github.com/betaflight/betaflight-configurator/issues/2498